### ISSUE TITLE
Don't raise an error when property ID is nil in development

### ISSUE THREAD

**Issue by tmaier** [danielbayerlein/middleman-google-analytics#19]
I configured google analytics the following way:

```
activate :google_analytics do |ga|
  ga.tracking_id = ENV['GOOGLE_ANALYTICS_ID']
  ga.anonymize_ip = true
  ga.debug = false
  ga.development = false
  ga.minify = true
end
```

`ENV['GOOGLE_ANALYTICS_ID']` is `nil` in development.

I get the following error message in development mode

```
Google Analytics: Please specify a property ID
/usr/local/opt/rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/middleman-google-analytics-1.0.2/lib/middleman-google-analytics/extension.rb:23:in `after_configuration': No property ID given (RuntimeError)
	from /usr/local/opt/rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/middleman-core-3.3.3/lib/middleman-core/extension.rb:130:in `block in bind_after_configuration'
```

**Comment by danielbayerlein** [comment]
Sorry for the delay! v1.1.0 is coming!

**Comment by danielbayerlein** [comment]
Fixed with v1.1.0!
